include/z64animation.h

```diff
@@ -10,7 +10,7 @@ struct Actor;
 struct SkelAnime;
 
 #define LINK_ANIMATION_OFFSET(addr, offset) \
-    (((u32)_link_animetionSegmentRomStart) + ((u32)addr) - ((u32)_link_animetionSegmentStart) + ((u32)offset))
+    (((uintptr_t)_link_animetionSegmentRomStart) + ((uintptr_t)(addr)) - ((uintptr_t)_link_animetionSegmentStart) + ((uintptr_t)(offset)))
```

Comment:
```suggestion
    (((uintptr_t)_link_animetionSegmentRomStart) + ((uintptr_t)(addr)) - ((uintptr_t)_link_animetionSegmentStart) + ((u32)(offset)))
```
Not sure I'd consider an offset a pointer so `uintptr_t` doesn't seem appropriate. Also it seems to match fine in its one use without a cast on the offset maybe just remove it?